.github/workflows/build-and-push-containers.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and push containers
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release-container:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Compile tag list
+        id: tags
+        run: |
+          TAG=${GITHUB_REF/refs\/tags\//}
+          PREFIX=ghcr.io/laminas/automatic-releases
+          MAJOR="${PREFIX}:$(echo ${TAG} | cut -d. -f1)"
+          MINOR="${MAJOR}.$(echo ${TAG} | cut -d. -f2)"
+          PATCH="${PREFIX}:${TAG}"
+          echo "::set-output name=tags::[\"${MAJOR}\",\"${MINOR}\",\"${PATCH}\"]"
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Not super-important, but some spacing (blank lines around blocks) would go a long way with improving readability

Reply:
Done!